curriculum/challenges/english/08-coding-interview-prep/data-structures/size-of-the-set.english.md

```diff
@@ -6,12 +6,12 @@ challengeType: 1
 
 ## Description
 <section id='description'>
-In this exercise we are going to create a size function for our Set. This function should be named <code>this.size</code> and it should return the size of the collection.
+  Set class should also have a <code>size()</code> method which returns the size of the collection.
```

Comment:
I would remove the word "also" here so the challenge is standalone.

Reply:
This is continuation of previous challenges in which the camper created "add" and "remove" methods.
Do you still think we should remove it?

Reply:
If this is a continuation from the previous challenge, I don't see the harm in adding the word `also`.  My only suggestion is to start the sentence with `The set` instead of `Set`.